apps/dashboard/src/components/primitives/container.tsx

```diff
@@ -0,0 +1,5 @@
+import { cn } from '../../utils/ui';
+
+export const Container = ({ children, className }: { children: React.ReactNode; className?: string }) => {
+  return <div className={cn('mx-auto w-full max-w-[1152px] px-14 py-14', className)}>{children}</div>;
+};
```

Comment:
Add a reusable container at 1152px width

Reply:
to me feels like to generic component with very specific CSS 😅

Reply:
The goal for this one is to be consistent across all pages that needs to use the container width. This is the first page to need it